plugins/presentation/src/plugin.ts

```diff
@@ -41,7 +41,7 @@ export default (platform: Platform, deps: { core: CoreService, i18n: I18n }): Pr
   const coreService = deps.core
   const i18nService = deps.i18n
 
-  platform.setResource(ui.component.ObjectBrowser, ObjectBrowser)
+  // platform.setResource(ui.component.ObjectBrowser, ObjectBrowser)
```

Comment:
Why we need two lines?

Reply:
It looks like we don't need `ObjectObserver` anymore, but im afraid to remove it, so I just keep it for a while. I'll undo commenting for it in `plugin.ts`